server/routes/person.ts

```diff
@@ -32,8 +33,20 @@ personRoutes.get('/:id', async (req, res, next) => {
   }
 });
 
+function filterMediaByType(media: any[], mediaType: MediaType | 'all'): any[] {
+  switch (mediaType) {
+    case 'movie':
+      return media.filter((result) => result.media_type === 'movie');
+    case 'tv':
+      return media.filter((result) => result.media_type === 'tv');
+    default:
+      return media;
+  }
+}
+
```

Comment:
You could do it all from the web app? You don't need to add an extra query param and filter the media on server-side, you can just filter the media on the client side.